tests/commands/test_new.py

```diff
@@ -0,0 +1,83 @@
+import subprocess
+import os
+import logging
+from pathlib import Path
+
+logging.basicConfig()
+logger = logging.getLogger('onyo')
+logger.setLevel(logging.INFO)
+
+test_dirs = ['simple',
+             's p a c e s',
+             's p a/c e s',
+             'r/e/c/u/r/s/i/v/e',
+             'relative',
+             'one',
+             'two',
+             'three',
+             'overlap/one',
+             'overlap/two',
+             'overlap/three',
+             'very/very/very/deep'
+             ]
+
+
+def populate_test_repo(path):
+
+    ret = subprocess.run(['onyo', 'init', path])
+    assert ret.returncode == 0
+
+    # enter repo
+    original_cwd = Path.cwd()
+    os.chdir(path)
+
+    # create dirs
+    ret = subprocess.run(['onyo', 'mkdir'] + test_dirs)
+    assert ret.returncode == 0
+
+    # return to home
+    os.chdir(original_cwd)
+
+
+def test_new_non_interactive():
+    populate_test_repo('top_level')
+    os.chdir('top_level')
+
+    # create new asset for all different folders
+    for i in range(0, len(test_dirs)):
+        input_str = 'laptop\napple\nmacbookpro\n' + str(i)
+        file = 'laptop_apple_macbookpro.' + str(i)
+        ret = subprocess.Popen(['onyo', 'new', '--non-interactive', test_dirs[i]],
```

Comment:
Is there a reason for using `Popen` here rather than `subprocess.run`? It's meant exactly for this use case. Less boilerplate and no need to use `communicate`.